### ISSUE TITLE
Need a newer SDK with fixed CoreGenerateSatelliteAssemblies

### ISSUE THREAD

**Issue by tmeschter** [dotnet/roslyn#23193]
If build using core MSBuild, there will be an error as following

```
/home/wul/roslyn/Binaries/Obj/BasicCodeAnalysis/Debug/cs/Microsoft.CodeAnalysis.VisualBasic.resources.cs(1,1): error CS1010: Newline in constant
```

This is due to https://github.com/dotnet/sdk/issues/1748. That issue tracks the work to fix the bug in the SDK; this bug will be used to track our adoption of an SDK with a fix in dotnet/roslyn.

**Comment by tmeschter** [comment]
Actually, while there may be problems due to `WriteCodeFragment` not escaping the code, the issue blocking localization in the dotnet/roslyn repo is different and more fundamental.

The `CoreGenerateSatelliteAssemblies` task always calls csc.exe to build the satellite assembly, but it generates a source file with the various assembly attributes (`AssemblyFileVerionAttribute`, etc.) _using the language of the project_. So in a VB project it will generate VB code into a .cs file, and then try to compile it with csc.exe. As you might expect that doesn't work very well.

**Comment by tmeschter** [comment]
I've filed issue https://github.com/dotnet/sdk/issues/1748 against dotnet/SDK. We'll use this issue to track adoption of a version of the SDK with the fix (when available) in the dotnet/roslyn repo.

**Comment by tmeschter** [comment]
Fixed in commit cd09eb908ed669e77ca90ab6413731ca8c1fc5e9.
